src/OSPSuite.UI/Controls/UxTreeView.cs

```diff
@@ -64,6 +64,7 @@ public UxTreeView()
          ToolTipForNode = node => node.ToolTip;
          ToolTipController = new ToolTipController();
          ToolTipController.GetActiveObjectInfo += (o, e) => ShowToolTip(e);
+         OptionsMenu.ShowExpandCollapseItems = false;
```

Comment:
this is a breaking change in new devexpress. This ensures old behavior everywhere